scheduler/generic_sched.go

```diff
@@ -68,6 +69,9 @@ const (
 	maxPastRescheduleEvents = 5
 )
 
+// minVersionMaxClientDisconnect is the minimum version that supports max_client_disconnect.
+var minVersionMaxClientDisconnect = version.Must(version.NewVersion("1.2.6"))
```

Comment:
@tgross Should I bump the version number in my branch to `1.3.0` or do we have a technique for faking this during development?

Reply:
Tricky... we don't bump `version.go` until the actual release so even if you swapped out your feature branch to 1.3.0 it'd be broken once we merged it to `main`. But the version we're developing on isn't `1.2.6`, it's `1.2.6-dev`. So we might be able to get away with setting this value to `1.2.6-dev` and then replacing it with `1.3.0` _after_ the release candidate ships.

It's been a very long time since we've had a feature that was version-checked like this. So it's worth double-checking for yourself that old check we wrote handles cases where we have a suffix like this anyways, so that ex. `1.3.0+ent` users don't end up with a broken implementation. 😀